storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -1128,6 +1161,11 @@ void Archived_Redo_Log_Monitor::thread_func() {
   }
 
   xb_mysql_query(mysql, "SELECT innodb_redo_log_archive_stop()", false);
+  if (!innodb_redo_log_archive) {
+    xb::info() << "innodb_redo_log_archive_dirs is not set,set it NULL .";
```

Comment:
This message is measleading, if it is not set already (I read this as NULL) why we set it to NULL again ?

Reply:
feels a little measleading ，the true meaning is  the server innodb_redo_log_archive_dirs is not set(NULL) and PXB innodb_redo_log_archive_dirs is not NULL .  in order to not change user option, so change  it to NULL at last.
 I will  correct  the mistake.